Math magicians components

tchasinga opened the pull request:
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
JavaScript requirements
Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
Project requirements
In the previous project, you worked on the setup of a React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html), now you should:

Delete all the boilerplate from Create React App (text, images, styles).
Create a new directory called components.
Inside components, create a new Calculator.js file.
In Calculator.js, create a React component that matches this design:

tchasinga committed:
first commits

tchasinga committed:
new updating

tchasinga committed:
new commit

tchasinga committed:
new updating

tchasinga committed:
new commit and organizing

tchasinga committed:
updating file

addod19 commented:
### Hi @tchasinga, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because the README file lacks relevant headers like the `future features`, the only option header to omit or remove from the README is the FAQ. Kindly take note. Here is the [template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

tchasinga committed:
code checking

Dear code viewer in my eslintrc have adding a simple function that help switching the prop-types , and that function help me fix the errors that i'd

joskalenda reviewed (approved):
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @tchasinga , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
| <img src="https://media2.giphy.com/media/2NpKgBuUZtXpu/giphy.gif"/> |

### Highlights
✅ ... linters
✅ ... Readme, Code

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
